README.md

```diff
@@ -14,35 +14,37 @@ In addition, using QKM, you can connect to your Ethereum nodes to sign your tran
 
 ## Run QKM
 
-Available docker images can be found at `docker.consensys.net/pub/quorum-key-manager`.
+First you have to write your manifest files and set the following environment variable:
+
+```bash
+export HOST_MANIFEST_PATH={your_manifests_folder}
+```
+ 
```

Comment:
Maybe we could add a like to some manifest exemples here ?

Reply:
Documentation about manifest is added in the official doc website only to avoid duplicity